composer/init.sls

```diff
@@ -0,0 +1,23 @@
+include:
+  - jcu.php
+  - jcu.php.json
+  - jcu.php.intl
+  - jcu.php.xml
+
+get-composer:
+  cmd.run:
+    - name: 'CURL=`which curl`; $CURL -sS https://getcomposer.org/installer | php'
```

Comment:
With installation like this, I try and record a specific URL / version / hash to ensure validated code gets installed (otherwise you're trusting whatever's available at that URL at the time). Have a look at https://github.com/jcu-eresearch/myregister-deployment/blob/75ec49de49134ecfc45eb58e5967d803b203268c/salt/roots/salt/myregister/init.sls#L79-L99 for inspiration; I did this once before